### PR TITLE
MINOR: create a checklist for testing a new Storm release

### PR DESCRIPTION
**Pull request by Ethanlm**
This creates a checklist so new devs have a better idea on reviewing a release

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -85,3 +84,24 @@ gpg --print-md SHA512 RELEASE_NOTES.html > RELEASE_NOTES.html.sha512
...
 3. Delete the git tag.
+
+# How to vote on a release candidate
+
+We encourage everyone to review and vote on a release candidate to make an Apache Storm release more reliable and trustworthy.
+
+Below is a checklist that one could do to review a release candidate. 
+Please note this list is not exhaustive and only includes some of the common steps. Feel free to add your own tests.
+
+1. Verify files such as *.asc, *.sha512;
```
Comment (srdo) [inline RELEASING.md]: I think we should link to the utility script in dev-tools/rc for this. I didn't know about it until I happened across it recently, would be good to raise visibility.

File: RELEASING.md [hunk 2]
```diff
@@ -85,3 +84,24 @@ gpg --print-md SHA512 RELEASE_NOTES.html > RELEASE_NOTES.html.sha512
 2. Delete the staged distribution files from https://dist.apache.org/repos/dist/dev/storm/
 
 3. Delete the git tag.
+
+# How to vote on a release candidate
```
Comment (srdo) [inline RELEASING.md]: I'm wondering if this belongs at the top of the file? The audience for it is probably broader than the audience for how to author a release.

Comment (Ethanlm) [reply RELEASING.md]: make sense

File: RELEASING.md (hunk 1 above)
Comment (Ethanlm) [reply RELEASING.md]: I think the original order is fine. We need to have release candidate then we can review and vote. It's more natural to me

Comment (srdo) [reply RELEASING.md]: I think your comments got switched somehow.

Ok, it's fine. I'd just like this guidance to be visible to people.

**Comment by srdo** [comment]
+1

**Pull request merged**